addon_imps/storage/dropbox.py

```diff
@@ -0,0 +1,113 @@
+import dataclasses
+import typing
+
+from addon_toolkit.interfaces import storage
+
+
+class DropboxStorageImp(storage.StorageAddonHttpRequestorImp):
+    """storage on dropbox.com
+    see https://www.dropbox.com/developers/documentation/http/documentation
+    """
+
+    async def get_external_account_id(self, _: dict[str, str]) -> str:
+        return ""
+
+    async def list_root_items(self, page_cursor: str = "") -> storage.ItemSampleResult:
+        if page_cursor:
+            async with self.network.POST(
+                "/files/list_folder/continue",
+                {"cursor": page_cursor},
+            ) as _response:
+                _parsed = _DropboxParsedJson(await _response.json_content())
+                return storage.ItemSampleResult(
+                    items=list(_parsed.item_results()),
+                )
```

Comment:
don't we need to return `next_sample_cursor` or `total_count` here

Reply:
Oh yeh, I missed it